src/components/footer/footer.module.scss

```diff
@@ -0,0 +1,13 @@
+@import "../../css/colors.scss";
+
+.container {
+  width: 100%;
+  justify-content: center;
+  display: flex;
+  position: relative;
```

Comment:
Is this prop necessary?

Reply:
Only the flex props are necessary, the rest I can eliminate.